core/kazoo_stdlib/src/kz_json.erl

```diff
@@ -264,30 +266,45 @@ from_list(PL) when is_list(PL) ->
     ?JSON_WRAPPER([{K, utf8_binary(V)} || {K, V} <- PL, V =/= 'undefined']).
 
 -spec from_list_recursive(json_proplist()) -> object().
-from_list_recursive([]) -> new();
-from_list_recursive(L)
+from_list_recursive(L) ->
+    from_list_recursive(L, #{}).
+
+-type from_list_options() :: #{no_ascii_list => boolean()
```

Comment:
I wish the name does not have to be no double negative like `ascii_list_enforced` something like that. Can it be `ascii_list_enforced` => true as default on line 280? Or a comment or update md what no_ascii_list means.